read-fonts/src/tables/postscript/charstring.rs

```diff
@@ -55,21 +55,21 @@ where
     P: Pen,
 {
     fn move_to(&mut self, x: Fixed, y: Fixed) {
-        self.0.move_to(x.to_f64() as f32, y.to_f64() as f32);
+        self.0.move_to(x.to_f32(), y.to_f32());
     }
 
     fn line_to(&mut self, x: Fixed, y: Fixed) {
-        self.0.line_to(x.to_f64() as f32, y.to_f64() as f32);
+        self.0.line_to(x.to_f32(), y.to_f32());
     }
 
     fn curve_to(&mut self, cx0: Fixed, cy0: Fixed, cx1: Fixed, cy1: Fixed, x: Fixed, y: Fixed) {
         self.0.curve_to(
-            cx0.to_f64() as f32,
-            cy0.to_f64() as f32,
-            cx1.to_f64() as f32,
-            cy1.to_f64() as f32,
-            x.to_f64() as f32,
-            y.to_f64() as f32,
+            cx0.to_f32(),
+            cy0.to_f32(),
+            cx1.to_f32(),
+            cy1.to_f32(),
+            x.to_f32(),
+            y.to_f32(),
```

Comment:
Nit: to_f32 would have been a nice standalone JMM PR. I wouldn't bother to hoist it out now tho.